modules/administration_manual/pages/installation/system_requirements.adoc

```diff
@@ -40,14 +40,8 @@ we officially recommend and support:
 
 [NOTE]
 ====
-ownCloud 10.1 requires a minimum php version of 7.1.
-
-If you use Ubuntu 16.04:
-
-* PHP 7.1 and 7.2 are only available via ppa. To add a ppa to your system, use this command: 
-`sudo add-apt-repository ppa:user/ppa-name`.
-* PHP 7.2 standard installable, but you have to install some mandatory modules yourself, like 
-http://php.net/manual/en/intro.intl.php[intl].
+ownCloud 10.1 requires a minimum PHP version of *5.6*.
+However, the next release, 11.0, requires a minimum PHP version of *7.1*.
```

Comment:
Please just remove both lines. The table above states the supported PHP versions which is correct until we announce deprecations. No need to give explicit version number statements, I think.

Reply:
Sorry, misclicked :)